components/table/table-col-sort-button.js

```diff
@@ -44,7 +45,7 @@ export class TableColSortButton extends LocalizeCoreElement(FocusMixin(LitElemen
 				type: String
 			},
 			/**
-			 * The type of data in the column. Used to set the title.
+			 * ACCESSIBILITY: The type of data in the column. Used to set the title.
```

Comment:
Similarly to my earlier comment, I think this might need an example or some further context, I found it hard to understand until I investigated.